examples/personal-data/client/components/Home.js

```diff
@@ -48,7 +48,11 @@ const Home = () => {
     }
   )
   useEffect(() => {
-    getUser()
+    if (window.location.hash === '#deleted') {
```

Comment:
It's better to use react-router's matching state instead direct browser host object